sdk/communication/azure-communication-email/src/main/java/com/azure/communication/email/EmailAsyncClient.java

```diff
@@ -117,4 +122,13 @@ PollerFlux<EmailSendResult, EmailSendResult> beginSend(EmailMessage message, Con
             TypeReference.createInstance(EmailSendResult.class),
             TypeReference.createInstance(EmailSendResult.class));
     }
+
+    void verifyRecipientEmailAddressesNotNull(List<EmailAddress> recipients) {
+        if (recipients != null) {
+            for (EmailAddress recipient : recipients) {
+                Objects.requireNonNull(recipient, "recipient 'EmailAddress' cannot be null.");
+                Objects.requireNonNull(recipient.getAddress(), "EmailAddress 'address' cannot be null.");
```

Comment:
This will throw NullPointerException if recipient is null for instance, right?

Reply:
Yup, that's correct.